docs/hub/models-uploading.md

```diff
@@ -6,6 +6,46 @@ You can link repositories with an individual, such as [osanseviero/fashion_brand
 
 There are several ways to upload models to the Hub, described below. We suggest adding a [Model Card](./model-cards) to your repo to document your model.
 
+## Using the Mixin class
+
+In case your model is a (custom) PyTorch model, the recommended way is to leverage the [`PyTorchModelHubMixin` [class](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin). It is a minimal class which adds `from_pretrained` and `push_to_hub` capabilities to any `nn.Module`. Here is how to use it:
```

Comment:
```suggestion
In case your model is a (custom) PyTorch model, the recommended way is to leverage the [`PyTorchModelHubMixin` class](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin). It is a minimal class which adds `from_pretrained` and `push_to_hub` capabilities to any `nn.Module`. Here is how to use it:
```